src/hooks/Recoil_Hooks.js

```diff
@@ -232,6 +233,11 @@ function useInterface(): RecoilInterface {
     function useRecoilState<T>(
       recoilState: RecoilState<T>,
     ): [T, SetterOrUpdater<T>] {
+      if (__DEV__) {
+        if (!isRecoilValue(recoilState))
+          throw new Error('No atom or selector provided');
```

Comment:
Hmm... maybe not clear that what was passed was invalid vs needing something else to be passed in - maybe "Invalid parameter, atom or selector expected"